### DISCUSSION TITLE
no life sign from AP GR6 & TD-MX after update to 3.0.1

### DISCUSSION THREAD

**Discussion by 3D-Cat** [FrSkyRC/ETHOS-Feedback-Community#4895] [Q&A]
First I did update my X20S to 3.0.1 TD-ISRM then updated my 2 AP GR6FB receivers to  3.0.1 without any problem.
But it didn't went well for the AP GR6 and TD-MX receivers. Although OTA update showed success there was no life sign from both receivers afterwards
Any idea how to recover the receivers.
thanks

**Comment by mgsanna** [comment]
Try to reflash instead of OTA, by wire.... and s.port directly trought the suite

**Reply by 3D-Cat** [reply]
tried already, it reports device not found or not ready. LED does not light up when power the receiver

**Comment by 3D-Cat** [comment]
all receivers are not usable anymore and frsky has no solution to recover them. CONGRATUALATIONS FRSKY good job

**Comment by Beastyboy80** [comment]
I updated my TD-MX today. I have to register and bind again. It works but I have no telemetry via F-Port

**Comment by 3D-Cat** [comment]
it looks like AP GR6 bootloader is corrupted after updating from V1.0.10 to V3.0.1 and as described it's no more functional
if someone knows a way to reflash the bootloader without sending the RX to service please let me know how
